src/main/jhipster/src/main/webapp/app/germplasm-details/genotyping/genotyping-pane.component.ts

```diff
@@ -169,16 +172,24 @@ export class GenotypingPaneComponent implements OnInit {
     }
 
     selectVariantsetOnChange() {
-        this.genotypingBrapiService.searchCallsets({
-            variantSetDbIds: [this.selectedVariantSet.variantSetDbId],
-            germplasmDbIds: [this.genotypingGermplasm.germplasmDbId]
-        }).subscribe((brapiResponse) => {
-            if (brapiResponse && brapiResponse.result.data.length) {
-                this.genotypingCallSet = brapiResponse.result.data[0];
-                this.loadGenotypingCalls();
-            } else {
-                this.alertService.error('genotyping.no.genotyping.callsets.found');
-            }
+        const exportFlapjackRequest = new ExportFlapjackRequest([], [], 'FLAPJACK', [this.germplasmSearchValue], true,
+            100, this.selectedVariantSet.referenceSetDbId);
+        this.genotypingBrapiService.exportFlapjack(exportFlapjackRequest).subscribe((response) => {
+            let file = response.replace('.fjzip', '').replace('/gigwaV2', '');
```

Comment:
@mderamos I think it's better to just extract the hostname from baseURL and combine it to the fileURL returned from Gigwa.

example:
let file = response.replace('.fjzip', '');
file = extractHostName(this.cropGenotypingParameter.baseUrl) + file;